### ISSUE TITLE
Configuration doubt with multiple django.po

### ISSUE THREAD

**Issue by sandroden** [nijel/weblate#1282]
Hi,
I have just installled weblate and it was a real breeze. Thanks.

Up to now I just used virtaal and never used web interface to *.po file. I'm trying to understand the setup of a Project. Let's take as example django itself. For each language there are several django.po file, one for each app under django.contrib. 

Should I use a single component for each app (that has it's own .pot) even if it comes from the same repository? I tried to do that and seems to work but the whole repository is cloned for any single component so I suspect it's not the proper way to go...

TIA
sandro
*:-)

**Comment by nijel** [comment]
Yes, there should be single component for every po file. To avoid cloning the repository multiple times use [Weblate internal URLs](https://docs.weblate.org/en/latest/vcs.html#weblate-internal-urls).

**Comment by nijel** [comment]
PS: Probably this is too hidden in the documentation as you're second person in last few months to ask this question (https://github.com/nijel/weblate/issues/1226). Where would you find it appropriate to mention this?

**Comment by sandroden** [comment]
The two places I'd suggest... do already have it! i.e.: help_text for component and explanation of the field.

So the reason I really didn't get it when I read it  is because at that point I didn't understand that a component can hold just one .po file and you're supposed to split project in as many components as there are .pot file. I'd suggest stressing that point would be beneficial for beginners and that would make it clear the use for internal repo url. A good point could be "translation organization" and at the beginning of "Component Configuration". Probably the misleadng sentence for me was: "Weblate automatically fetches the VCS and finds all matching translatable files."

Another point: the help_text of filemask as an example uses local/*/LC_MESSAGES/django.po that very rarely is a path starting from the root of the project. So I got it for a pattern and that led me to think it could refer to several different positions. I started thinking it was not when I realized that there was space for just one .pot file. But you already correctly explain that in the docs...

That said let me say that I consider that weblate documentation is a really well done peace of work! 

Thanks again
sandro
*:-)
